config/search/config.json

```diff
@@ -238,10 +207,6 @@
           "^https://data.vlaanderen.be/ns/dossier#behandelt",
           "http://mu.semte.ch/vocabularies/ext/publicatie/publicatieNummer"
         ],
-        "publicationFlowNumac": [
-          "^https://data.vlaanderen.be/ns/dossier#behandelt",
-          "http://mu.semte.ch/vocabularies/ext/publicatie/numacNummer"
-        ],
         "publicationFlowRemark": [
```

Comment:
Waarom gaat dit er uit?
Ze willen hier toch op zoeken in de nieuwe publicatieflow?

Reply:
Ja, klopt, maar waarschijnlijk is dat omwille van discussie over kardinaliteit daarrond terug uit de frontend verdwenen. Momenteel niet in gebruik https://github.com/kanselarij-vlaanderen/kaleidos-frontend/search?q=publicationFlowNumac&type=code . Waarschijnlijk komt het er later in andere hoedanigheid terug in ...